tpchgen/src/generators.rs

```diff
@@ -844,7 +845,7 @@ impl<'a> SupplierGeneratorIterator<'a> {
             s_address: self.address_random.next_value(),
             s_nationkey: nation_key,
             s_phone: self.phone_random.next_value(nation_key),
-            s_acctbal: self.account_balance_random.next_value() as f64 / 100.0,
+            s_acctbal: Decimal(self.account_balance_random.next_value() as i64),
```

Comment:
I do wonder if using an i32 as the internal representation would be more efficient (though I see total price is an i64)

Reply:
Probably would be, but figured if there's a difference, it's likely tiny. I was originally writing this with i32, but decided to switch to i64 to reduce the likelihood of me introducing a bug. It's easier to cast everything to i64, and the i64 -> i32 casts are just from well-defined min/max bounds that fit in i32.